packages/cf-deploy-config-inquirer/src/index.ts

```diff
@@ -0,0 +1,55 @@
+import { getQuestions } from './prompts';
+import type {
+    CfDeployConfigPromptOptions,
+    CfDeployConfigQuestions,
+    CfSystemChoice,
+    CfDeployConfigAnswers
+} from './types';
+import { promptNames } from './types';
+import { initI18nCfDeployConfigInquirer } from './i18n';
+import type { InquirerAdapter } from '@sap-ux/inquirer-common';
+import autocomplete from 'inquirer-autocomplete-prompt';
+import { ToolsLogger, type Logger } from '@sap-ux/logger';
+import LoggerHelper from './logger-helper';
+
+/**
+ * Retrieves Cloud Foundry deployment configuration prompts.
+ *
+ * This function returns a list of cf deployment questions based on the provided application root and prompt options.
+ *
+ * @param {CfDeployConfigPromptOptions} promptOptions - The configuration options for prompting during cf target deployment.
+ * @param logger - The logger instance to use for logging.
+ * @returns {Promise<CfDeployConfigQuestions[]>} A promise that resolves to an array of questions for cf target prompting.
+ */
+async function getPrompts(
+    promptOptions: CfDeployConfigPromptOptions,
+    logger?: Logger
+): Promise<CfDeployConfigQuestions[]> {
+    LoggerHelper.logger = logger ?? new ToolsLogger({ logPrefix: '@sap-ux/abap-deploy-config-inquirer' });
```

Comment:
Logger name needs to be udated.

Reply:
You could update this code as follows as the `ToolsLogger` is already set in `LoggerHelper`;
```
    if (logger) {
        LoggerHelper.logger = logger;
    }
```